src/test/resources/features/successful-endpoints.feature

```diff
@@ -123,6 +123,16 @@ Feature: Successful rest calls
       | Carlos2 |
     And response json path element "$[0].id" should be "1"
 
+  Scenario: Retrieve users list cleaning db
+    Given I have no rows in the "models" table
```

Comment:
I would add here something like:

```gherkin
Given I have the following rows in the "models" table
| | | |
When I have no rows in the "models" table
Then I make a GET ....
And response should be json:
"""
[]
"""
```

to verified that we are actually removing the existing rows from the db